src/project-manager.ts

```diff
@@ -729,6 +730,25 @@ export class InMemoryFileSystem implements ts.ParseConfigHost, ts.ModuleResoluti
 		this.path = path;
 		this.overlay = new Map<string, string>();
 		this.rootNode = { file: false, children: new Map<string, FileSystemNode>() };
+		// adding library files from the local file system
+		this.initTypeScriptLibraries();
+	}
+
+	isTypeScriptLibrary(path: string): boolean {
+		return this.typeScriptLibraries.has(util.toUnixPath(path));
+	}
+
+	/**
+	 * Collects TypeScript library files
+	 */
+	private initTypeScriptLibraries() {
```

Comment:
This is doing sync IO when the InMemoryFileSystem is created, which is done for every new connection and will then block anything happening in other connections. Before it was done on startup, which was better.